guides/templates/guides/match_email.txt

```diff
@@ -82,6 +82,10 @@ Finally, please let us if there is any other way in which we can
 support your interactions.  Please mail any questions you have to
 lead@guides.ietf.org.
 
+{% if match.participant.remote == "YES" %}NOTE: {{ match.participant.given_name }} will be remote for this IETF.{% endif %}
```

Comment:
This is going to leave a lot of extra whitespace when the conditions are false.
If you want to improve that later, move the linefeeds inside the if statements.